projects/dendrites/permutedMNIST/experiments/__init__.py

```diff
@@ -20,13 +20,12 @@
 # ----------------------------------------------------------------------
 
 from .base import CONFIGS as BASE
+from .hyperparameter_search import CONFIGS as HYPERPARAMETERSEARCH
 from .batch import CONFIGS as BATCH
 from .batch_mnist import CONFIGS as BATCH_MNIST
```

Comment:
This init file has kept all imports in alphabetical order, please keep it that way!

Reply:
solved with Isort